libs/langchain/langchain/agents/openai_assistant/base.py

```diff
@@ -527,7 +530,7 @@ def _get_response(self, run: Any) -> Any:
             if all(
                 (
                     isinstance(content, openai.types.beta.threads.TextContentBlock)
-                    if openai.version.VERSION.startswith("1.14")
+                    if int(openai_major_version) > 1 or int(minor_version >= 14)
```

Comment:
Should be an and rather than an or?

Reply:
> Should be an and rather than an or?

Done.

Reply:
if major version is greater than 1 then it doesn't matter what minor version is